rfcs/0000_cortical_messaging_protocol_cmp_v1.md

```diff
@@ -136,7 +136,7 @@ All in-transit Cortical Messages to be received by Goal State Selectors are deli
 
 #### 2.1.7 Motor Modules
 
-During the Motor Modules phase the Cortical Messages from Goal State Selectors are delivered to the Motor Modules. The Motor Modules process the Cortical Messages and output actions. The phase ends when all actions are created.
+During the Motor Modules phase the Cortical Messages from Goal State Selectors are delivered to the Motor Modules. The Motor Modules process the Cortical Messages, observations, and proprioceptive state and generate zero, one, or more actions. The phase ends when all actions are created.
```

Comment:
This is currently the case (although it happens in the dataloader/motor system since we don't have motor modules yet) but with the changes to SMs to be able to send goal states I would like to get to the point where any sensory processing happens in the SM and none in the motor system (except potentially some fast reflex loops for security purposes).